src/core/Table/Table.tsx

```diff
@@ -88,6 +91,19 @@ export type TableProps<
    * @default 300
    */
   intersectionMargin?: number;
+  /**
+   * A function that will be used for rendering a component for each row if that row is expanded.
+   * Component will be placed right after the row, in the same row group div.
+   */
+  subComponent?: (row: Row<T>) => React.ReactNode;
+  /**
+   * Handler for row expand events. Will trigger when expanding and condensing rows.
+   */
+  onExpand?: (state: TableState<T>) => void;
```

Comment:
Add expanded rows data like we do with `onSelect`.

Reply:
So which data do we want to represent? Should we return row ids for that? For me it would make sense at least.

Reply:
I would say expanded rows data (`row.original`) array.